ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9vertex.cpp

```diff
@@ -33,12 +33,110 @@ D3DDECLUSAGE usage_types[] = { D3DDECLUSAGE_POSITION, D3DDECLUSAGE_COLOR, D3DDEC
   D3DDECLUSAGE_BLENDINDICES, D3DDECLUSAGE_DEPTH, D3DDECLUSAGE_TANGENT, D3DDECLUSAGE_BINORMAL, D3DDECLUSAGE_FOG, D3DDECLUSAGE_SAMPLE };
 
 map<int, LPDIRECT3DVERTEXBUFFER9> vertexBufferPeers;
+map<int, LPDIRECT3DINDEXBUFFER9> indexBufferPeers;
```

Comment:
Are these actually used in other files? If not, please use an anonymous namespace instead of namespace `enigma`.

Reply:
No, they are not used in other files, at all, in the OpenGL implementation either. I can do this, but may I ask exactly why you think that matters? What is the reason?

Reply:
There's no sense polluting any namespace (not just `enigma_user` or the global namespace) if you don't need to. Putting this in an anonymous namespace is the same as making it static—it prevents linker errors if two files declare a variable like this, and it lets maintainers know that it's safe to delete these variables if they're no longer used in this file.

Reply:
Done: c793972138d3ebd892626b73078d7ddb8a79d2bc